pyk/src/pyk/cterm.py

```diff
@@ -15,11 +14,25 @@ class CTerm:
     constraints: Tuple[KInner, ...]
 
     def __init__(self, term: KInner) -> None:
-        config, constraint = splitConfigAndConstraints(term, Sorts.GENERATED_TOP_CELL)
+        config, constraint = CTerm._split_config_and_constraints(term)
         constraints = CTerm._normalize_constraints(flattenLabel('#And', constraint))
         object.__setattr__(self, 'config', config)
         object.__setattr__(self, 'constraints', constraints)
 
+    @staticmethod
+    def _split_config_and_constraints(kast: KInner) -> Tuple[KInner, KInner]:
```

Comment:
As a first step, I'd just move this to `cterm.py`. Later, when we are ready to eliminate calls to the function, e.g. by replacing

```python
config, constraint = split_config_and_constraints(term)
```

by

```python
config, *constraints = Cterm(term)
constraint = mlAnd(constraints)
```

we can make it a private static method.

Reply:
Done.